src/commands/Tools/Dictionary/urban.ts

```diff
@@ -16,43 +17,54 @@ export default class extends SkyraCommand {
 			nsfw: true,
 			requiredPermissions: ['EMBED_LINKS'],
 			runIn: ['text'],
-			usage: '<query:string> [page:integer{0,10}]',
-			usageDelim: '#'
+			usage: '<query:string>'
 		});
 	}
 
-	public async run(message: KlasaMessage, [query, ind = 1]: [string, number]) {
-		const index = ind - 1;
-		if (index < 0) {
-			throw message.language.tget('RESOLVER_POSITIVE_AMOUNT');
-		}
+	public async run(message: KlasaMessage, [query]: [string]) {
+		const response = await message.sendEmbed(
+			new MessageEmbed({ description: message.language.tget('SYSTEM_LOADING'), color: getColor(message) || 0xFFAB2D })
+		) as KlasaMessage;
 
-		const { list } = await fetch(`https://api.urbandictionary.com/v0/define?term=${encodeURIComponent(query)}`, 'json') as UrbanDictionaryResultOk;
+		let { list } = await fetch(`https://api.urbandictionary.com/v0/define?term=${encodeURIComponent(query)}`, 'json') as UrbanDictionaryResultOk;
 
-		const result = list[index];
-		if (typeof result === 'undefined') {
-			throw index === 0
-				? message.language.tget('COMMAND_URBAN_NOTFOUND')
-				: message.language.tget('COMMAND_URBAN_INDEX_NOTFOUND');
-		}
+		list = list.sort((a, b) => b.thumbs_up - b.thumbs_down - (a.thumbs_up - a.thumbs_down));
```

Comment:
Instead of mutating the variable, I would prefer the first variable to be a constant called `result`, and then do

```typescript
const list = result.list.sort((a, b) => b.thumbs_up - b.thumbs_down - (a.thumbs_up - a.thumbs_down));
```